core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token_fr.properties

```diff
@@ -0,0 +1,13 @@
+authenticate-security-token.getting.started=D\u00e9marrage
```

Comment:
@batmat I had the same question and noticed all translated files are using unicode characters. I thought this was to help prevent someone to commit accidently with a bad encoding so I stuck with existing convention (2016, still annoyed with encoding...).
Thank you for the remark, it confirmed my analysis.